share/wake/lib/core/boolean.wake

```diff
@@ -24,20 +24,32 @@ export data Boolean =
 # Unary operator for Boolean NOT.
 # !True  = False
 # !False = True
-export def !x = if x then False else True
+export def !(x: Boolean): Boolean =
+    if x then
+        False
+    else
+        True
```

Comment:
(lol)

Reply:
Yeah, something about expanding the oneliner to match the style guide makes the humor of the definition stand out even more.  Or am I missing something?

Reply:
There's nothing wrong with the change. It's just a funny function, made more funny by being 4 lines long.